vulnerabilities/models.py

```diff
@@ -326,6 +327,26 @@ class Advisory(models.Model):
     into structured data
     """
 
+    def save(self, *args, **kwargs):
+        hash = ""
+        if self.summary:
+            hash = hashlib.md5(self.summary.encode("utf-8")).hexdigest()
+        json_fields = [self.affected_packages, self.references]
+        # Case 1: a ="Hello" and b="World"
+        # Case 2: a ="HelloWo" and b="rld"
+        # hash(a) + hash(b) for both the cases will give same result
+        # therefore hash( hash(a) + hash(b) ) is better than hash(a + b)
+        for json_field in json_fields:
+            if json_field:
+                hash = hashlib.md5(
+                    (hash + hashlib.md5(json.dumps(json_field).encode("utf-8")).hexdigest()).encode(
+                        "utf-8"
+                    )
+                ).hexdigest()
+        self.unique_content_id = hash
+        super(Advisory, self).save(*args, **kwargs)
```

Comment:
This is likely enough?
```suggestion
        super().save(*args, **kwargs)
```